web/server/codechecker_server/skiplist.py

```diff
@@ -0,0 +1,103 @@
+# -------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -------------------------------------------------------------------------
+
+"""Parse the skip list file."""
```

Comment:
I would highlight that a similar file can be found in the analyzer part of the CodeChecker.

For example:
```py
"""
Parse the skip list file.

NOTE: There is a similar file in the analyzer part.
Please consider to update that in case of changing this.
"""
```